dev-docs/source/RunningTheUnitTests.rst

```diff
@@ -103,31 +103,33 @@ Starting in your build folder (e.g. Mantid/Code/debug):
 
    -  Not possible with ctest.
 
-Visual Studio/ XCode note
-#########################
+Running Unit Tests from Within Visual Studio
+############################################
 
-In Visual Studio the user can alter the properties of the subset of
-tests (inside the unitTest directory (e.g. AlgorithmTest). In the
-properties box it is possible to specify a specific test to run by
-typing its name in the TargetName box. Then to execute the test, right
-click the subset of tests and select debug and then start new instance.
+Open the Mantid solution in Visual Studio 2015. To run a subset of tests (for example ``UnitTests/AlgorithmsTest``);
 
-To run the tests under one of these environments then you will need to
-open a command window and change to the build directory. Once there you
-can run the tests by selecting the configuration;
+-  In the Solution Explorer, right click the top level directory for the tests (in this case ``UnitTests/AlgorithmsTest``) and select Properties.
```

Comment:
Things like ``UnitTests/AlgorithmsTest`` are generally referred to as projects and not directories - can we use this term instead?